docker/build/simulation.Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ros:galactic-ros-base
+FROM osrf/ros:humble-desktop
```

Comment:
Any reason for changing the base image? On my local tests the simulation failed to start with the following error:

```
...
[ERROR] [1671569485.535183450] [lifecycle_manager_localization]: Server map_server was unable to be reached after 4.00s by bond. This server may be misconfigured.
[ERROR] [1671569485.535283812] [lifecycle_manager_localization]: Failed to bring up all requested nodes. Aborting bringup.
```

However, by using `ros:humble-ros-base` as the base image I was able to get the simulation up and running with no errors

Reply:
I chose `osrf/ros:humble-desktop` as the base because Nav2 doesn't work well on my side with the `ros:humble-ros-base` somehow. 
I just updated it to the one that works for you since we don't use gazebo for testing normally.